SwanCustomEnvironments/swancustomenvironments/scripts/builders/conda.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Create environment and install packages
+${BUILDER_VERSION} env create -f ${REQ_PATH} -n ${ENV_NAME} -y
+
+# With a symlink on the user's home, no other customenvs can be created
+ln -f -s /opt/conda/envs/${ENV_NAME} ${ENV_PATH}
+ENV_PATH="/opt/conda/envs/${ENV_NAME}"
```

Comment:
Why is changing `ENV_PATH` necessary?

Reply:
ENV_PATH needs to be change due to the explanation above, the conda env is not under `/home/$user` anymore, so we need to point it to the new actual path

Reply:
I believe after my suggestion for the command, the conda env can be under /home/$user, so you do not need to symlink nor update ENV_PATH.